### PR TITLE
Update to Roslyn 2.0.0-beta3

### PR DESCRIPTION
**Pull request by DustinCampbell**
Note: I removed the references to Roslyn from src/OmniSharp.Plugins/project.json. I'm not sure why those were there.

### CODE REVIEW

File: src/OmniSharp.Roslyn.CSharp/project.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
   "dependencies": {
     "OmniSharp.Abstractions": "1.0.0",
     "OmniSharp.Roslyn": "1.0.0",
-    "Microsoft.CodeAnalysis.CSharp": "1.3.0",
-    "Microsoft.CodeAnalysis.CSharp.Features": "1.3.0",
-    "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.0"
+    "Microsoft.CodeAnalysis.CSharp": "2.0.0-beta3",
+    "Microsoft.CodeAnalysis.CSharp.Features": "2.0.0-beta3",
```
Comment (david-driscoll) [inline src/OmniSharp.Roslyn.CSharp/project.json]: I'm assuming the internal class used for find metadata hasn't changed, but do you know if it has?

File: src/OmniSharp.Plugins/project.json [hunk 2]
```diff
@@ -5,12 +5,7 @@
   },
   "dependencies": {
     "OmniSharp.Abstractions": "1.0.0",
-    "Newtonsoft.Json": "8.0.3",
-    "Microsoft.CodeAnalysis": "1.3.0",
```
Comment (david-driscoll) [inline src/OmniSharp.Plugins/project.json]: I can't think why these would be here... probably copy and paste fail.

**Comment by david-driscoll** [comment]
Wonder if I should turn off LGTM now that github has native approvals... probably should, lol.

**Comment by DustinCampbell** [comment]
yup :smile:

File: src/OmniSharp.Roslyn.CSharp/project.json (hunk 1 above)
Comment (DustinCampbell) [inline src/OmniSharp.Roslyn.CSharp/project.json]: I assume there were tests? :smile:

**Comment by david-driscoll** [comment]
Turned on approvals required, turned off the lgtm webhook.

File: src/OmniSharp.Roslyn.CSharp/project.json (hunk 1 above)
Comment (DustinCampbell) [inline src/OmniSharp.Roslyn.CSharp/project.json]: I assume there were tests?

Comment (david-driscoll) [inline src/OmniSharp.Roslyn.CSharp/project.json]: Yeah... there were... so everything should be okay, lol.  (I just woke up not long ago!)

Comment (DustinCampbell) [inline src/OmniSharp.Roslyn.CSharp/project.json]: I just took a look at the following files in Roslyn, that OmniSharp reflects into. It looks like they're OK.

* [CSharpMetadataAsSourceService.cs](https://github.com/dotnet/roslyn/blob/master/src/Features/CSharp/Portable/MetadataAsSource/CSharpMetadataAsSourceService.cs) - Last updated 9 months ago
* [MetadataAsSourceHelpers.cs](https://github.com/dotnet/roslyn/blob/master/src/Features/Core/Portable/MetadataAsSource/MetadataAsSourceHelpers.cs) - Recent changes were to update resource string IDs. Before that it was 9 months.
* [SymbolKey.cs](https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Core/Portable/SymbolKey/SymbolKey.cs) - This has more significant changes but it looks like the API we use [here](https://github.com/OmniSharp/omnisharp-roslyn/blob/dev/src/OmniSharp.Roslyn/MetadataHelper.cs#L81-L87) is essentially [the same](https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Core/Portable/SymbolKey/SymbolKey.cs#L123-L126).

File: src/OmniSharp.Plugins/project.json (hunk 2 above)
Comment (DustinCampbell) [inline src/OmniSharp.Plugins/project.json]: That's what I was thinking

**Comment by david-driscoll** [comment]
LGTM CURSES!

**Pull request merged**